connections/ethereum/connection.go

```diff
@@ -165,6 +165,34 @@ func (c *Connection) SafeEstimateGas(ctx context.Context) (*big.Int, error) {
 	}
 }
 
+func (c *Connection) SafeEstimateGasTipCap(ctx context.Context, baseFee *big.Int) (*big.Int, *big.Int, error) {
+	if c.maxGasPrice.Cmp(baseFee) < 0 {
```

Comment:
i think we should pay baseFee any way dont we? I mean there is no reason to check it, since its chain based and is obligatory

Reply:
I put this in because of the check and logic I made with `gasFeeCap` and `maxGasPrice` that you commented on below. If the `maxGasPrice` is less than the baseFee we would end up with a negative tip and too low of `gasFeeCap`. (https://github.com/ChainSafe/ChainBridge/pull/690/files#diff-349cc0f2c50c806250ed335c780d7a6dd9b7ae1cd01fed7e8cfacfa16bba81f6R190). I could instill set it so we just pay the `baseFee` plus like a tiny priority fee of like 1 gwei. Greg linked below to how `web3.js` implements no specified `gasFeeCap` and they do it this way.

Reply:
Yes lets do it this way. If maxGasPrice < baseFee, then maxGasPrice should be baseFee + 1 gwei, because stopping entire bridge is not the case i guess. 
Also can you please rename variables  gasFeeCap => maxFeePerGas, tip => maxPriorityFeePerGas, bcs its easy to get lost in such range of variables names